index.html

```diff
@@ -16,8 +16,32 @@
 </head>
 <body>
   <header>
+    <h2 class="container">Code Journal</h2>
   </header>
   <main>
+    <div class="container" data-view="entry-form">
+      <form>
+        <h2>New Entry</h2><br />
+        <div class="row">
+          <div class="column-half">
+          <img src="images/placeholder-image-square.jpg">
+          </div>
+        <div class="column-half">
+          <label for="title">Title:</label><br />
+          <input type="text" id="new-post-title" name="title" required><br />
```

Comment:
Change the `id` here. It should describe that this is the `input` for the title, not the title itself.